src/studio/src/designer/backend/Factories/ModelFactory/JsonMetadataParser.cs

```diff
@@ -88,7 +90,8 @@ private void CreateModelFromMetadataRecursive(Dictionary<string, string> classes
                     }
                     else
                     {
-                        classBuilder.AppendLine("    [XmlElement(\"" + element.Value.XName + "\")]");
+                        classBuilder.AppendLine("    [XmlElement(\"" + element.Value.XName + "\", Order = " + elementOrder + ")]");
+                        elementOrder = elementOrder + 1;
```

Comment:
A tiny improvement in my eyes would be to change the order of these statements. The idea is to use the value being incremented on the same iteration as the increment. Like in an regular for loop.

Reply:
Done